index.html

```diff
@@ -134,13 +134,44 @@ <h2>Tonic</h2>
         <li><img src="images/list2.png" alt="list2"></li>
         <li><img src="images/list3.png" alt="list3"></li>
       </ul>
+      <div class="finalframe">
+        <img src="images/language frame.png" alt="frame photo">
+      </div>
+      <div class="skillframe">
+        <img src="images/skillframe.png" alt="skill text">
+      </div>
     </div>
-    <div class="finalframe">
-      <img src="images/language frame.png" alt="frame photo">
-    </div>
-    <div class="skillframe">
-      <img src="images/skillframe.png" alt="skill text">
-    </div>
+  
+    
+    <div class="contact-me">
+      <div class="shapes">
+
+      <h1 class="contat">Contact me</h1>
+      <div>
+          <p class="para">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      </div>
+
+    <form action="https://formspree.io/f/{form_id}" method="post">
+      <div class="form">
```

Comment:
- [ ] As requested by the previous reviewer you should create an account on Formspree and use it. Here is the link for the lesson [Link](https://formspree.io/html/)